Directory.Build.props

```diff
@@ -1,8 +1,10 @@
 <Project>
-    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-        <NoWarn>CS1998</NoWarn>
-    </PropertyGroup>
-    <PropertyGroup>
-        <LangVersion>latest</LangVersion>
-    </PropertyGroup>
-</Project>
\ No newline at end of file
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>CS1998</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
```

Comment:
Default is true anyways - cf https://learn.microsoft.com/de-de/dotnet/core/project-sdk/msbuild-props#enablenetanalyzers

Reply:
Yes, but for > .NET 5 only.
`Gandalan.IDAS.WebApi.Client.Wpf` is .NET Framework project.